src/uct/api/uct.h

```diff
@@ -272,6 +275,18 @@ enum uct_progress_types {
 
 /**
  * @ingroup UCT_AM
+ * @brief Flags for active message send operation.
+ */
+enum {
+    UCT_AM_FLAG_SIGNALED = UCS_BIT(0)  /**< Trigger @ref UCT_EVENT_RECV_SIG_AM
+                                            event on remote side, and on best effort
+                                            avoid trigerring @ref UCT_EVENT_RECV_AM event.
```

Comment:
Trigger @ref UCT_EVENT_RECV_SIG_AM event on remote side. Make best effort attempt to avoid triggering @ref UCT_EVENT_RECV_AM event.